spec/account_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'account'
+
+describe Account do
+  
+  let(:account) { Account.new([], 2000) }
+  let(:entry) { double(:entry) }
+  let(:second_entry) { double(:entry) }
```

Comment:
It's really important to format code idiomatically.  Try using a linter to help with this.